force-app/main/default/lwc/updatePaymentMethodModal/updatePaymentMethodModal.js

```diff
@@ -8,15 +8,46 @@ const TAB_KEY_CODE = 9;
 const TAB_KEY_STRING = "Tab";
 
 export default class UpdatePaymentMethodModal extends LightningElement {
+    
+    @api isBankPaymentAllowed;
+
     @api openUpdatePaymentMethod;
+    
     @api currentRecord;
 
+    @track
+    paymentMethodValue = "";
+
     labels = {
       updatePaymentMethod
     }   
 
+    @track
+    paymentMethodOptions = [
+        { label: 'Credit Card', value: 'Credit Card' }
+    ];
+
+
     renderedCallback() {
-      this.template.addEventListener("keydown", (e) => this.handleKeyUp(e));
+        this.template.addEventListener("keydown", (e) => this.handleKeyUp(e));
+        if(this.currentRecord){
+            this.determineACHpaymentMethodAndAddAsOption();
+            this.paymentMethodValue = this.currentRecord.paymentMethod;
+        }
+    }
+
+    determineACHpaymentMethodAndAddAsOption(){
+        if(this.isBankPaymentAllowed){
```

Comment:
Couldn't these all be combined into one `if` condition? It looks like the body of each `if` block is the same logic.

Reply:
done!